<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>733</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="labelPlate">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>400</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>渝BAG993(蓝色)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDraw">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>开始抽奖</string>
   </property>
   <property name="text">
    <string>开  始</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelLuckyPlate">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>幸运车牌号（1/100)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDrawActivity">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>440</width>
     <height>47</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>重庆高速2017安全文明积极参与者抽奖活动</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>741</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>抽奖结果</string>
   </property>
   <widget class="QLabel" name="labelLuckyPlates">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>701</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>渝BAG993(蓝色)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="buttonDraw_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>开始抽奖</string>
   </property>
   <property name="text">
    <string>抽 奖 结 果</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
